assets/performanceprofile/scripts/set-rps-mask.sh

```diff
@@ -33,4 +33,6 @@ function find_dev_dir {
 [ -d "${dev_dir}" ] || { sleep 5; find_dev_dir; }  # search failed, wait a little and try again
 [ -d "${dev_dir}" ] || { echo "${dev_dir}" directory not found >&2 ; exit 0; } # the interface disappeared, not an error
 
-echo "${mask}" > "${dev_dir}"/queues/rx-0/rps_cpus
+for i in "${dev_dir}"/queues/rx-*/rps_cpus; do
```

Comment:
`i` is not a great name here but considering how costly the testing is, I won't resubmit *just* for this. But let's keep in mind if you need to resubmit. Let's discuss the name, something like `rps_file` `mask_file` or the liking.

Reply:
the only real concern I have is: this is bound to break if this path changes. How likely is this to change?

Reply:
I don't know maybe @pabeni can tell us

Reply:
Well the alternative solution based on find works the same way, so I am not worried about this part.

Reply:
I was just wondering if the quotes should be extended to cover the whole path.

Reply:
> Well the alternative solution based on find works the same way, so I am not worried about this part.

fair enough

Reply:
> I was just wondering if the quotes should be extended to cover the whole path.

We had the same with find and it worked

Reply:
> the only real concern I have is: this is bound to break if this path changes. How likely is this to change?

do you mean the sysfs path/dir names? they are part of uAPI, are expected to be as stable as possible. Any change to that would be a serious kernel bug.

Reply:
>do you mean the sysfs path/dir names? they are part of uAPI, are expected to be as stable as possible. Any change to that would be a serious kernel bug.

Thank you for your feedback, if that's the case I think we can move along with the current approach.